<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CriarPessoa</class>
 <widget class="QDialog" name="CriarPessoa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEndereco">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Cpf:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTelefone">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCpf">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineIdade">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNome">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Idade:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Endereco:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>90</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineNome</tabstop>
  <tabstop>lineCpf</tabstop>
  <tabstop>lineEndereco</tabstop>
  <tabstop>lineIdade</tabstop>
  <tabstop>lineTelefone</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
